cogs/remind_me.py

```diff
@@ -154,21 +158,30 @@ async def autocomplete_get_delays(ctx: TeXBotAutocompleteContext) -> set[str]:
                     if month < 10:
                         delay_choices.add(f"0{month}{joiner}{year}")
 
-        elif match := re.match(r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2})\Z", ctx.value):  # noqa: E501
+        elif match := re.match(
+            r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2})\Z",
+            ctx.value,
+        ):
```

Comment:
`if` selection statements that cannot be split into boolean flags (like here where many flags would need to be defined at the top of the if-else stack) should just have overrunning max line lengths ignored, and should not be split onto multiple lines